src/main/drivers/max7456.c

```diff
@@ -202,8 +202,8 @@ uint16_t maxScreenSize = VIDEO_BUFFER_CHARS_PAL;
 // screenBuffer with shadowBuffer to upgrade only changed chars.
 // This solution is faster then redrawing entire screen.
 
-static uint8_t screenBuffer[VIDEO_BUFFER_CHARS_PAL+40]; // For faster writes we use memcpy so we need some space to don't overwrite buffer
-static uint8_t shadowBuffer[VIDEO_BUFFER_CHARS_PAL];
+static uint16_t screenBuffer[VIDEO_BUFFER_CHARS_PAL];
```

Comment:
extra bit may be saved separately using bit manipulation, considerably reducing memory usage.
`uint32_t screenBufferExt[(VIDEO_BUFFER_CHARS_PAL + sizeof(uint32_t) - 1) / sizeof(uint32_t));`